### PR TITLE
Document tests in Python.

### PR DESCRIPTION
**Pull request by JDarDagran**
Signed-off-by: Jakub Dardzinski <kuba0221@gmail.com>

### CODE REVIEW

File: docs/development/developing/python-tests.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Python tests
```
Comment (mobuchowski) [inline docs/development/developing/python-tests.md:2]: I think this should be separate page for each integration. 

Also, would be nice to explain how to set up required dependencies.

File: docs/development/developing/python-tests.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Additionally, outside of `tox` you should run `mypy` static code analysis. You can do that with:
+```
+python -m mypy openlineage
+```
+
+### Integration tests
+Integration tests are located in `tests/integration/tests` directory. They require running Docker containers to provision local test environment: Airflow components (worker, scheduler), databases (PostgreSQL, MySQL) and OpenLineage events consumer.
+
+#### How to run
+Integration tests require usage of _docker compose_. There are scripts prepared to make build images and run tests easier.
```
Comment (mobuchowski) [inline docs/development/developing/python-tests.md:50]: Would be nice to explain how to run multiple consecutive tests, eg. after some changes.

File: docs/development/developing/python-tests.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+`setup.cfg` contains tox-related configuration. By default `tox` command runs:
+1. `flake8` linting
+2. `pytest` command
+
+Additionally, outside of `tox` you should run `mypy` static code analysis. You can do that with:
+```
+python -m mypy openlineage
+```
+
+### Integration tests
```
Comment (mobuchowski) [inline docs/development/developing/python-tests.md:46]: Would be nice to provide example how to add example test. For example, expanding on adding new custom extractor.

File: docs/development/developing/python-tests.md (hunk 2 above)
Comment (JDarDagran) [reply docs/development/developing/python-tests.md:50]: To run consecutive tests we probably need to add that possibility in scripts. Or you have something else in mind?

File: docs/development/developing/python-tests.md (hunk 3 above)
Comment (JDarDagran) [reply docs/development/developing/python-tests.md]: Done.

File: docs/development/developing/python-tests.md (hunk 2 above)
Comment (JDarDagran) [reply docs/development/developing/python-tests.md]: This take additional action in the OL code. I'll open PR to document this after development.

**Review by mobuchowski — APPROVED** [review]

**Pull request merged**